Task/Tasks/TSKSelectorTask.h

```diff
@@ -77,7 +93,7 @@
      May not be NULL.
  @result A newly initialized TSKSelectorTask instance with the specified name, target, and action.
  */
-- (instancetype)initWithName:(NSString *)name target:(id)target selector:(SEL)selector;
+- (instancetype)initWithName:(nullable NSString *)name target:(id)target selector:(SEL)selector;
```

Comment:
I don’t think we support nullable targets. Why do you think it should be nullable? We don’t send it up the responder chain or anything.